docroot/modules/custom/sitenow_events/css/single-event.css

```diff
@@ -50,10 +50,13 @@
   font-weight: bold;
 }
 .date-instance__past {
-  color: #999;
+  color: #737373;
```

Comment:
Should we be using a variable for this?

Reply:
It's not configured to be compiled in a workspace. Maybe we wait for https://github.com/uiowa/uiowa/issues/4916 for that?

Reply:
Do we not just have a CSS variable for this value?

Reply:
Nope, we need to switch everything from SCSS variables to CSS variables.

Reply:
Added https://github.com/uiowa/uids/issues/985 as a reminder to start that process.